Parsers/ASimFileEvent/Parsers/vimFileEventSentinelOne.yaml

```diff
@@ -0,0 +1,161 @@
+Parser:
+  Title: File Event Parser for SentinelOne
+  Version: '0.1.0'
+  LastUpdated: Sep 18, 2023
+Product:
+  Name: SentinelOne
+Normalization:
+  Schema: FileEvent
+  Version: '0.2.1'
+References:
+- Title: ASIM File Event Schema
+  Link: https://aka.ms/ASimFileEventDoc
+- Title: ASIM
+  Link: https://aka.ms/AboutASIM
+- Title: SentinelOne Documentation
+- Link: https://<SOneInstanceDomain>.sentinelone.net/api-doc/overview
+Description: |
+  This ASIM parser supports normalizing SentinelOne logs to the ASIM File Event normalized schema. SentinelOne events are captured through SentinelOne data connector which ingests SentinelOne server objects such as Threats, Agents, Applications, Activities, Policies, Groups, and more events into Microsoft Sentinel through the REST API.
+ParserName: vimFileEventSentinlOne
```

Comment:
@jayeshprajapaticrest  there's spelling error here. pls fix.

Reply:
Done

Reply:
there is one more in ASIM
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/0cfa94be-91e9-4e76-b43a-1535911b0ec5)

Reply:
Done